gui/browserv7/src/RBrowser.cxx

```diff
@@ -693,8 +693,21 @@ void ROOT::Experimental::RBrowser::WebWindowCallback(unsigned connid, const std:
 
       fWebWindow->Send(connid, res);
    } else if (arg.compare(0,7, "DBLCLK:") == 0) {
-      auto str = ProcessDblClick(arg.substr(7));
-      if (str.length() > 0) fWebWindow->Send(connid, str);
+
+      if (arg.at(8) != '[') {
```

Comment:
Is is necessary here? 
Do you have situation when options not send and just itemname is provided?

Reply:
I think so, without it an editor request which look like this : 
`DBLCLK:ROOTConfig.cmake`
and a root request which look like : 
`DBLCLK: ["tutorials/hsimple.root/hpx;1","hist"]`
so on the line :
`auto arr = TBufferJSON::FromJSON<std::vector<std::string>>(arg.substr(7));`
With the editor there isn't an array, or even a drawing option

Reply:
Agree, dblclick may be handled in other situations too